spotify-conversion-next/package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "next lint"
```

Comment:
When you see changes to `package.json`, that means I installed new packages with `npm install ...`

which means when you pull down this branch, you need to run "npm install" to get the packages on your local machine. This file just tells your machine which packages need to be pulled.